test/JDBC/expected/babel_datatype_sqlvariant-vu-cleanup.out

```diff
@@ -27,5 +25,9 @@ drop sequence babel_datatype_sqlvariant_vu_prepare_t2_sec;
 go
 drop sequence babel_datatype_sqlvariant_vu_prepare_t4_sec;
 go
+~~ERROR (Code: 3732)~~
+
+~~ERROR (Message: cannot drop sequence babel_datatype_sqlvariant_vu_prepare_t4_sec because other objects depend on it)~~
```

Comment:
There is error while dropping sequence. Please fix this.
Why have you removed the line above?
```
drop table babel_datatype_sqlvariant_vu_prepare_t4;

Reply:
Done